pkg/cmd/workspace/create.go

```diff
@@ -108,6 +114,29 @@ var CreateCmd = &cobra.Command{
 			}
 			visited[*projects[i].Source.Repository.Url] = true
 			projects[i].EnvVars = getEnvVariables(&projects[i], profileData)
+
+			if builderFlag == string(create.DEVCONTAINER) || devcontainerPathFlag != "" {
```

Comment:
Since `--multi-project` flag invalidates builder flags, does it make sense to move these if's to `processCmdArguments()` instead?